src/sdk/model/common/snapshot/has-snapshot.ts

```diff
@@ -0,0 +1,32 @@
+/* istanbul ignore next: autogenerated */
```

Comment:
should not be above imports, should be just above class declaration

Reply:
fixed